sci-libs/silo/silo-4.11.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit fortran-2
+
+DESCRIPTION="A mesh and field I/O library and scientific database"
+HOMEPAGE="https://wci.llnl.gov/simulation/computer-codes/silo"
+SRC_URI="https://wci.llnl.gov/sites/wci/files/2021-09/${P}-bsd.tgz"
+S="${WORKDIR}/${P}-bsd"
+
+SLOT="0"
+LICENSE="BSD"
+KEYWORDS="~amd64 ~x86"
+IUSE="browser +hdf5 +silex +szip +qt5 test"
+RESTRICT="!test? ( test )"
+
+REQUIRED_USE="silex? ( qt5 )"
+
+RDEPEND="
+	szip? ( virtual/szip )
+	hdf5? ( sci-libs/hdf5 )
+	qt5? ( dev-qt/qtgui:5 )
+"
+
+DEPEND="${RDEPEND}"
+
+PATCHES=(
+	"${FILESDIR}/${P}-autoreconf.patch"
```

Comment:
Small nit, I prefer `"${FILESDIR}"/${P}-autoreconf.patch` (with only FILESDIR in quotes), but no need to change if you don't want to